coalib/testing/LocalBearTestHelper.py

```diff
@@ -169,26 +194,17 @@ def check_results(self,
                               list,
                               msg='The given results are not a list.')
 
-        if results in [[], ()]:
-            msg = ("The local bear '{}' yields a result although it "
-                   "shouldn't.".format(local_bear.__class__.__name__))
-            check_order = True
-        else:
-            msg = ("The local bear '{}' doesn't yield the right results."
-                   .format(local_bear.__class__.__name__))
-            if check_order:
-                msg += ' Or the order may be wrong.'
```

Comment:
ahhh or wait, we can maybe move it for now to `assertComparableObjectsEqual` into the first `else` clause, above `self.assertEqual(observed_result, expected_result)`. We can then talk about deleting that again in Phase2. This way we keep the old behaviour :D